packages/good-design/src/core/web3/KimaModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useCallback } from "react";
+import { useModal } from "../../hooks/useModal";
+import { Box, Text } from "native-base";
+import { LearnButton } from "../buttons";
+
+type BridgeNetworks = {
+  origin: string;
+  destination: string;
+};
+
+interface KimaModalProps {
+  success: boolean | undefined;
+  networks: BridgeNetworks;
+  resetState: any;
+  children: any;
+}
+
+const KimaModalCopy = (success: boolean, networks: BridgeNetworks) => ({
```

Comment:
define as 2 components: KimaModalHeader / KimaModalBody receiving such props

Reply:
1 component, KimaModalHeader

Chose to do learn button directly as I didn't see the need for it to be a separate component if there are no props and/or deps